pkg/trace/api/api.go

```diff
@@ -451,6 +457,50 @@ func (r *HTTPReceiver) handleStats(w http.ResponseWriter, req *http.Request) {
 	r.statsProcessor.ProcessStats(in, req.Header.Get(headerLang), req.Header.Get(headerTracerVersion))
 }
 
+// handleConfig handles config request.
+func (r *HTTPReceiver) handleConfig(w http.ResponseWriter, req *http.Request) {
+	defer timing.Since("datadog.trace_agent.receiver.config_process_ms", time.Now())
+	ts := r.tagStats(v06, req.Header)
+	tags := ts.AsTags()
+	statusCode := http.StatusOK
+	defer func() {
+		tags = append(tags, fmt.Sprintf("status_code:%d", statusCode))
+		metrics.Count("datadog.trace_agent.receiver.config", 1, tags, 1)
```

Comment:
```suggestion
		metrics.Count("datadog.trace_agent.receiver.config_request", 1, tags, 1)
```
Could end up ambiguous otherwise...